src/ReTestItems.jl

```diff
@@ -61,6 +61,27 @@ struct TimeoutException <: Exception
     msg::String
 end
 
+_is_good_nthread_str(str) = occursin(r"^(auto|[1-9]\d{0,4})$", str)
+_validate_nworker_threads(n::Int) = n > 0 ? string(n) : throw(ArgumentError("Invalid value for `nworker_threads` : $n"))
+function _validate_nworker_threads(str)
```

Comment:
i had a quick look to see how Julia itself does this. I was hoping it'd be something nicer, but... https://github.com/JuliaLang/julia/blob/229269badb899cfcb52ad3e2103058560cec5573/src/jloptions.c#L478